specification/communication/resource-manager/Microsoft.Communication/stable/2020-08-20/CommunicationService.json

```diff
@@ -0,0 +1,1118 @@
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List by subscription": {
+            "$ref": "./examples/listBySubscription.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Communication/communicationServices": {
+      "get": {
+        "description": "Handles requests to list all resources in a resource group.",
+        "operationId": "CommunicationService_ListByResourceGroup",
+        "summary": "List By Resource Group",
+        "tags": [
+          "CommunicationService"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The response describes the list of CommunicationServices in a resourceGroup.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationServiceResourceList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "List by resource group": {
+            "$ref": "./examples/listByResourceGroup.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Communication/communicationServices/{communicationServiceName}": {
+      "patch": {
+        "description": "Operation to update an existing CommunicationService.",
+        "operationId": "CommunicationService_Update",
+        "summary": "Update",
+        "tags": [
+          "CommunicationService"
+        ],
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "Parameters for the update operation",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/TaggedResource"
```

Comment:
TaggedResource? Not CommunicationServiceResource?

Reply:
I'm not sure of the reason for this. Tagging in @bviglietta who might have an answer as to why.

Reply:
I'm not sure why it's TaggedResource, since it does seem like CommunicationServiceResource would make more sense here. Although CommunicationServiceResource doesn't actually have any mutable fields that you could patch, so it doesn't make a huge difference.

Reply:
@allenjzhang Is this blocking our signoff? This part hasn't changes since our previous version.

Reply:
If it hasn't changed, I won't block the PR. But please review with team whether it is correct. Thx

Reply:
@allenjzhang This has now been updated to use CommunicationServiceResource instead of TaggedResource